packages/app/src/app/pages/common/Modals/NetlifyLogs/index.tsx

```diff
@@ -1,50 +1,51 @@
-import React, { FunctionComponent, useState, useEffect } from 'react';
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { Button } from '@codesandbox/common/lib/components/Button';
+
 import { useOvermind } from 'app/overmind';
 
-import { Button } from '@codesandbox/common/lib/components/Button';
+import { Explanation, Heading } from '../elements';
 import { Container } from '../LiveSessionEnded/elements';
-import { Heading, Explanation } from '../elements';
 
-import { List, Item } from './elements';
+import { Item, List } from './elements';
 
-const NetlifyLogs: FunctionComponent = () => {
-  const [logs, setLogs] = useState(['Contacting Netlify']);
+export const NetlifyLogs: FunctionComponent = () => {
   const {
+    actions: { modalClosed },
     state: {
-      deployment: { netlifyLogs: url },
+      deployment: { netlifyLogs: netlifyLogsUrl },
     },
-    actions: { modalClosed },
   } = useOvermind();
+  const [logs, setLogs] = useState(['Contacting Netlify']);
```

Comment:
```suggestion
  const [logs, setLogs] = useState(['Waiting for build to start...']);
```

With the improved indication of build not started, below, we should be more explicit about what it is waiting for. It is not waiting for "contact" with Netlify, it is waiting for the build to start